activation/post.go

```diff
@@ -258,11 +226,11 @@ func (mgr *PostSetupManager) StartSession(opts atypes.PostSetupOpts, commitmentA
 			log.String("provider", fmt.Sprintf("%d", opts.ComputeProviderID)),
 		)
 
-		if err := newInit.Initialize(); err != nil {
+		if err := newInit.Initialize(mgr.ctx); err != nil {
 			mgr.mu.Lock()
 			defer mgr.mu.Unlock()
 
-			if errors.Is(err, initialization.ErrStopped) {
+			if errors.Is(err, context.Canceled) {
 				mgr.logger.Info("post setup session stopped")
```

Comment:
How about changing the log to say "session was stopped" to reflect that it was interrupted by an external event and not just finished?
```suggestion
				mgr.logger.Info("post setup session was stopped")
```

Reply:
I'm not sure what you mean. This message is logged when the caller calls `StopSession()`. "was stopped" to me implies it might have been unintentional, while this (at least as it is implemented at the moment) is always intentional.

Reply:
For me, it's the opposite :)
- something stopped - possibly on its own ("a car stopped")
- something **was** stopped - it was stopped by someone ("a car was stopped by the driver")